portfolio/src/main/webapp/script.js

```diff
@@ -11,6 +11,10 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+function initialize() {
```

Comment:
let's move development of the maps API into a new PR branch to keep things clean. let's look to finish up the original PR and get that merged in.